adserver/models.py

```diff
@@ -1476,6 +1476,18 @@ class BaseImpression(TimeStampedModel, models.Model):
 
     date = models.DateField(_("Date"), db_index=True)
 
+    # Decisions are a superset of all Offers.
+    # Every API request that comes in results in a Decision,
+    # and an Offer is only created when we actually offer an ad.
+    decisions = models.PositiveIntegerField(
```

Comment:
I'm more worried about this migration, since it adds a field to all our indexes. Hopefully it should be fast tho, and only impact reporting.